src/node.h

```diff
@@ -730,8 +822,36 @@ class NODE_EXTERN CommonEnvironmentSetup {
       MultiIsolatePlatform* platform,
       std::vector<std::string>* errors,
       EnvironmentArgs&&... env_args);
+  template <typename... EnvironmentArgs>
+  static std::unique_ptr<CommonEnvironmentSetup> CreateWithSnapshot(
```

Comment:
`CreateFromSnapshot` sounds a bit better?

Reply:
:+1: Done in a136c282a9e454929dd6265bf35afefa97591f97